### PR TITLE
Fetch and display all events

### PR DESCRIPTION
**Pull request by aidenm1**
[//]: # "Feel free to customize this template to your project's vibes"

### :croissant: Description
[//]: # "Required - Describe what's new in this PR in a few lines. A description and bullet points for specifics will suffice."

Fetch events from Supabase database and display on events page

### :baguette_bread: Screenshots
[//]: # "Required for frontend changes, otherwise optional but strongly recommended. Add screenshots of expected behavior - GIFs if you're feeling fancy! Use the provided image template. Drag the desired image into the PR, then copy the link into the placeholder."

image

## :sandwich: How to review
[//]: # 'Required - Describe the order in which to review files and what to expect when testing locally. Is there anything specifically you want feedback on? Should this be reviewed commit by commit, or all at once? What are some user flows to test? What are some edge cases to look out for?'

Review api/supabase/queries/events.ts, then types/schema.d.ts, then app/events/page.tsx

## :pie: Next steps
[//]: # "Optional - What's NOT in this PR, doesn't work yet, and/or still needs to be done. Note any temporary fixes in this PR that should be cleaned up later."

### :pretzel: Online sources
[//]: # 'Copy links to any tutorials or documentation that was useful to you when working on this PR'

### :bagel: Related PRs
[//]: # "Add related PRs you're waiting on/ PRs that will conflict, etc; if this is a refactor, feel free to add PRs that previously modified this code"

CC: @celinechoiii

### CODE REVIEW

File: api/supabase/queries/events.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import supabase from '../createClient';
+
+export async function getAllEvents() {
```
Comment (david-qing) [inline api/supabase/queries/events.ts]: change to fetchAllEvents for consistency

File: api/supabase/queries/events.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  const { data, error } = await supabase.from('events').select('*');
+
+  if (error) {
+    console.error('An error occurred trying to read events:', error);
+    return;
```
Comment (david-qing) [inline api/supabase/queries/events.ts]: maybe consider explicitly returning null here; might clear up some confusion

**Comment by celinechoiii** [comment]
Everything else looks good to me!! Great job on your first sprint 🥳
